src/common/components/paymentMethods/creditCardForm/creditCardForm.component.js

```diff
@@ -55,7 +55,8 @@ class CreditCardController {
         cardNumberPlaceholder: this.paymentMethod['card-number'],
         cardholderName: this.paymentMethod['cardholder-name'],
         expiryMonth: this.paymentMethod['expiry-month'],
-        expiryYear: parseInt(this.paymentMethod['expiry-year'])
+        expiryYear: parseInt(this.paymentMethod['expiry-year']),
+        securityCode: ''
```

Comment:
I don't think this matters if we are removing it from the DOM. It won't be used anywhere so maybe toss it.